lambdas/services/data_collection_service.py

```diff
@@ -195,29 +212,33 @@ def get_organisation_data(
 
         organisation_data_for_all_ods_code = [
             OrganisationData(
-                date=self.today_date,
+                date=start_time.strftime("%Y%m%d"),
                 **organisation_data_properties,
             )
             for organisation_data_properties in joined_query_result
         ]
 
         return organisation_data_for_all_ods_code
 
-    def get_application_data(self) -> list[ApplicationData]:
-        user_id_per_ods_code = self.get_active_user_list()
+    def get_application_data(
+        self, start_time: datetime, end_time: datetime
+    ) -> list[ApplicationData]:
```

Comment:
Were any of these service methods unit tested originally? Seems a number of functions have been changed but no tests have been updated